api/v1beta1/loadbalancerset_types.go

```diff
@@ -4,6 +4,13 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	// CreationTimeoutAnnotation can be set to override the defaut duration of
+	// 10 minutes, after which a new LoadBalancerMachine is deleted (by the LBM)
+	// if it reports no conditions yet.
+	CreationTimeoutAnnotation = "yawol.stackit.cloud/creationTimeout"
```

Comment:
Should be moved to `loadbalancermachine_types.go` as it can only be set on LBMs.